code/_compile_options.dm

```diff
@@ -12,7 +12,7 @@
 #define DATUMVAR_DEBUGGING_MODE
 
 ///Used to find the sources of harddels, quite laggy, don't be surpised if it freezes your client for a good while
-//#define REFERENCE_TRACKING
+#define REFERENCE_TRACKING
```

Comment:
Should we turn this back so it doesn't lag the server?

Reply:
Yeah, absolutely, which is why its a draft and all